Lombiq.Hosting.MultiTenancy.Tenants.Tests.UI/Lombiq.Hosting.MultiTenancy.Tenants.Tests.UI.csproj

```
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

```suggestion
    <TargetFramework>net6.0</TargetFramework>
    <DefaultItemExcludes>$(DefaultItemExcludes);.git*;node_modules\**</DefaultItemExcludes>
```

You can find this in this document: https://lombiq.atlassian.net/wiki/spaces/DEV/pages/19496967/Creating+a+new+Visual+Studio+project+or+solution